src/components/CmsFilters.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <!-- Filters -->
+  <div v-if="filters.length > 0" class="filters-section row">
+    <div class="full-width container pb-1">
+      <div class="categories-container">
+        <div v-for="category in filters" :key="category" class="categories">
+          <div
+            :style="setCategoryStyle(category)"
+            class="category-pill"
+            @click="selectCategory(category)"
+          >
+            <div class="category-pill-text">
+              {{ category }}
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop } from 'vue-property-decorator';
+
+import cmsClient from '../cmsHttp';
+
+@Component({
+  name: 'cms-filters',
+})
+export default class CmsFilters extends Vue {
+  @Prop(String) public zoneId!: string;
+
+  public filters: string[] = [];
+  public showFilters: boolean = false;
+  public selectedCategory: string | null = null;
```

Comment:
Maybe we should make `selectedCategory` non-null and have it default to the str `'All'`? I think since null and 'All' would have the same behavior, it might just be a little simpler, and we can do this now since we are guaranteed to always have one category selected.

Reply:
i agree